cpp/examples/abm_minimal.cpp

```diff
@@ -0,0 +1,153 @@
+    // Add hospital and ICU with 5 maximum contacs.
+    auto hospital = world.add_location(mio::abm::LocationType::Hospital);
+    world.get_individualized_location(hospital).get_infection_parameters().set<mio::abm::MaximumContacts>(5);
+    auto icu = world.add_location(mio::abm::LocationType::ICU);
+    world.get_individualized_location(icu).get_infection_parameters().set<mio::abm::MaximumContacts>(5);
+    // Add one supermarket, maximum constacts are assumed to be 20.
+    auto shop = world.add_location(mio::abm::LocationType::BasicsShop);
+    world.get_individualized_location(shop).get_infection_parameters().set<mio::abm::MaximumContacts>(20);
+    // At every school, the maximum contacts are 20.
+    auto school = world.add_location(mio::abm::LocationType::School);
+    world.get_individualized_location(school).get_infection_parameters().set<mio::abm::MaximumContacts>(20);
+     // At every workplace, maximum contacts are 10.
+    auto work = world.add_location(mio::abm::LocationType::Work);
+    world.get_individualized_location(work).get_infection_parameters().set<mio::abm::MaximumContacts>(10);
+
+    // People can get tested at work (and do this with 0.5 probability).
+    auto testing_min_time = mio::abm::days(1);
+    auto probability      = 0.5;
+    auto start_date       = mio::abm::TimePoint(0);
+    auto end_date         = mio::abm::TimePoint(0) + mio::abm::days(60);
+    auto test_type        = mio::abm::AntigenTest();
+    auto test_at_work = std::vector<mio::abm::LocationType>{mio::abm::LocationType::Work};
+    auto testing_criteria_work =
+        std::vector<mio::abm::TestingCriteria>{mio::abm::TestingCriteria({}, test_at_work, {})};
+    auto testing_scheme_work =
+        mio::abm::TestingScheme(testing_criteria_work, testing_min_time, start_date, end_date, test_type, probability);
+    world.get_testing_strategy().add_testing_scheme(testing_scheme_work);
+
+    // Assign locations to the people
+    for (auto& person : persons) {
+        //assign shop and event
+        person.set_assigned_location(event);
+        person.set_assigned_location(shop);
+        //assign hospital and ICU
+        person.set_assigned_location(hospital);
+        person.set_assigned_location(icu);
+        //assign work/school to people depending on their age
+        if (person.get_age() == mio::abm::AgeGroup::Age5to14) {
+            person.set_assigned_location(school);
+        }
+        if (person.get_age() == mio::abm::AgeGroup::Age15to34 || person.get_age() == mio::abm::AgeGroup::Age35to59) {
+            person.set_assigned_location(work);
+        }
+    }
+
+    // During the lockdown, social events are closed for 90% of people. 
+    auto t_lockdown = mio::abm::TimePoint(0) + mio::abm::days(10);
+    mio::abm::close_social_events(t_lockdown, 0.9, world.get_migration_parameters());
+
+    auto t0         = mio::abm::TimePoint(0);
+    auto tmax       = mio::abm::TimePoint(0) + mio::abm::days(30);
```

Comment:
Consider changing to 60 days here or 30 days in the test definition.

Reply:
I now change the testing period to 30 days.